go/router/templates/router/includes/actions.html

```diff
@@ -65,7 +65,7 @@
             Activating: Attempt to stop
         </button>
         <button
-         class="btn btn-danger action"
+         class="btn action"
          data-action="stop"
          data-url="{% router_screen router 'stop' %}"
          disabled>
```

Comment:
Coolio. I was trying to crowbar it in as relevant "because routing," but you are right. :)